### ISSUE TITLE
Issue templates

### ISSUE THREAD

**Issue by Wolfr** [smartcoop/design#114]
<img width="676" alt="Screen Shot 2021-10-08 at 21 54 23" src="https://user-images.githubusercontent.com/12690/136617867-328d198e-26ae-4f2a-a3b3-2691e6d67a40.png">

It might be cool to have issue templates. Any thoughts?

(From this [article](https://medium.com/societe-generale-design/taming-design-system-chaos-66bcadbf43e1))

**Comment by thusc** [comment]
Yes, issue templates are a great idea, especially if they assign some tags automatically.
